library/include/rocblas-functions.h

```diff
@@ -3026,6 +3026,308 @@ ROCBLAS_EXPORT rocblas_status rocblas_zgemv_strided_batched(rocblas_handle
                                                             rocblas_stride                stridey,
                                                             rocblas_int batch_count);
 
+/*! \brief BLAS Level 2 API
+
+    \details
+    trmv performs one of the matrix-vector operations
+
+         x = A*x or x = A**T*x,
+
+    where x is an n element vector and A is an n by n unit, or non-unit, upper or lower triangular matrix.
+
+    The vector x is overwritten on b.
```

Comment:
'b' is referred to in multiple places but not defined.

Reply:
!

Reply:
See commit